### DISCUSSION TITLE
TenderReference value

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#698] [Q&A] [answered]
What value should the TenderReference node contain? Looking at the production examples we see that it is a numerical value that has no relationship with other nodes. Is it the internal identifier of the offer in the system from which the shipment was made?

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/6af68839-f34c-4043-a57c-312c40b81155)

![image](https://github.com/OP-TED/eForms-SDK/assets/115476802/a83b20c5-1e73-4dbe-afb0-52c3ce1d2448)

Thanks in advance,
Alex

**Comment by YvesJo** [comment]
The information highlighted with a yellow background is the Tender Identifier (BT-3201) which is a Business Identifier (in opposition to the Technical Identifier OPT-321, TEN-xxxx)

**Reply by AlexAS81** [reply]
What does it mean to be a Business Identifier?

Could it be a database identifier from our platform, for example?

**Reply by YvesJo** [reply, marked as answer]
Exactly, a similar situation as with the Internal Identifier (BT-22) for Lots
